fling/dataset/domainnet.py

```diff
@@ -0,0 +1,93 @@
+import torch.utils.data as data
+import torchvision.transforms as transforms
+import os
+from PIL import Image
+import numpy as np
+
+from fling.utils.registry_utils import DATASET_REGISTRY
+
+
+@DATASET_REGISTRY.register('domainnet')
+class DomainNetDataset(data.Dataset):
+    r"""
+    Implementation for DomainNet dataset. 
+    """
+    def __init__(self, cfg: dict, domain: str,  train: bool): # cfg can be loaded
+        super(DomainNetDataset, self).__init__()
+        self.train_transform = transforms.Compose([
```

Comment:
Do not add data transform like this. Use the default config form. Refer to other dataset files.